joomla/joomla-cms
# [4.0] Namespace User - Terms plugin

### Summary of Changes

Namespaces User - Terms and Conditions plugin.

### Testing Instructions

Delete `libraries/autoload_psr4.php` file.
Enable User Registration.
Enable User - Terms and Conditions plugin.
Open registration form in frontend.
Check that Terms & Conditions field appears correctly in the form.


### Documentation Changes Required

IDK.
